hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotUtils.java

```diff
@@ -96,42 +95,18 @@ public static void dropColumnFamilyHandle(
   public static void checkSnapshotActive(OzoneManager ozoneManager,
                                          String snapshotTableKey)
       throws IOException {
-    checkSnapshotActive(getSnapshotInfo(ozoneManager, snapshotTableKey));
+    checkSnapshotActive(getSnapshotInfo(ozoneManager, snapshotTableKey), false);
   }
 
-  public static void checkSnapshotActive(SnapshotInfo snapInfo)
+  public static void checkSnapshotActive(SnapshotInfo snapInfo,
+                                         boolean override)
```

Comment:
```suggestion
                                         boolean skipCheck)
```

Reply:
Tho it sounds a bit weird to have a param to skip the check for the check method. If we fix the caller, how many places do we have to change? If not many we should do that instead.

Reply:
I initially thought about this, but it looks like we should be changing in many places. I kept on getting Exceptions from multiple request.